packages/instant/src/containers/connected_account_payment_method.ts

```diff
@@ -49,7 +55,24 @@ const mergeProps = (
     onInstallWalletClick: connectedDispatch.onInstallWalletClick,
     walletName: connectedState.providerState.name,
     onUnlockWalletClick: () => {
-        connectedDispatch.unlockWalletAndDispatchToStore(connectedState.providerState);
+        const isMobile = envUtil.isMobileOperatingSystem();
+        if (!isMobile) {
```

Comment:
what's going on here? -- it looks like we're accounting for a situation in which we display the wallet as 'locked' if they are on mobile, even if they are not on a mobile dapp app like coinbase wallet?

Reply:
Whoops was in a rush when I wrote this -- this is wrong.